res/gamedata/configs/text/ukr/openxray.xml

```diff
@@ -0,0 +1,171 @@
+		<text>³������</text>
+	</string>
+	<string id="st_opt_windowed_borderless">
+		<text>³������ ��� �����</text>
+	</string>
+	<string id="st_opt_fullscreen">
+		<text>�������������</text>
+	</string>
+	<string id="st_opt_fullscreen_borderless">
+		<text>������������� (������� Alt+Tab)</text>
+	</string>
+	<string id="ui_mm_r2_fxaa">
+		<text>FXAA</text>
+	</string>
+	<string id="ui_mm_r2_smap_size">
+		<text>����� ���� ����</text>
+	</string>
+	<string id="ui_mm_r3_dynamic_wet_surfaces_opt">
+		<text>���������� ���������</text>
+	</string>
+	<string id="ui_mm_r3_dynamic_wet_surfaces_near">
+		<text>��������� �������</text>
+	</string>
+	<string id="ui_mm_r3_dynamic_wet_surfaces_far">
+		<text>��������� �������</text>
+	</string>
+	<string id="ui_mm_r3_dynamic_wet_surfaces_sm_res">
+		<text>����� ���������</text>
+	</string>
+	<string id="ui_mm_fov">
+		<text>���� ����</text>
+	</string>
+	<string id="ui_mm_localization">
+		<text>����</text>
+	</string>
+	<string id="ui_mm_gamepad_setup">
+		<text>������������ ��������</text>
+	</string>
+	<string id="ui_mm_fps_limit">
+		<text>��������� FPS � ��</text>
+	</string>
+	<string id="ui_mm_fps_limit_in_menu">
+		<text>��������� FPS � ����</text>
+	</string>
+	<string id="ui_mm_shader_preset">
+		<text>����� �������</text>
+	</string>
+	<string id="ui_mm_shader_preset_vanilla">
+		<text>����������</text>
+	</string>
+	<string id="ui_mm_shader_preset_enhanced_default">
+		<text>��������� (��������)</text>
+	</string>
+	<string id="ui_mm_shader_preset_enhanced_high">
+		<text>��������� (������)</text>
+	</string>
+	<string id="ui_mm_color_grading_preset">
+		<text>�������� �������</text>
+	</string>
+	<string id="ui_mm_color_grading_default">
+		<text>����������</text>
+	</string>
+	<string id="ui_mm_color_grading_cold">
+		<text>�������</text>
+	</string>
+	<string id="ui_mm_color_grading_filmic_01">
+		<text>Filmic 1</text>
+	</string>
+	<string id="ui_mm_color_grading_filmic_02">
+		<text>Filmic 2</text>
+	</string>
+	<string id="ui_mm_color_grading_filmic_03">
+		<text>Filmic 3</text>
+	</string>
+	<string id="ui_mm_color_grading_hollywood">
+		<text>Hollywood</text>
+	</string>
+	<string id="ui_mm_color_grading_vanilla">
+		<text>����������</text>
+	</string>
+	<string id="ui_mm_color_grading_vibrant">
+		<text>Vibrant</text>
+	</string>
+	<string id="ui_mm_color_grading_warm">
+		<text>�����</text>
+	</string>
+	<string id="eng">
+		<text>��������� (English)</text>
+	</string>
+	<string id="fra">
+		<text>���������� (French)</text>
+	</string>
+	<string id="ger">
+		<text>ͳ������ (German)</text>
+	</string>
+	<string id="ita">
+		<text>��������� (Italian)</text>
+	</string>
+	<string id="pol">
+		<text>�������� (Polish)</text>
+	</string>
+	<string id="rus">
+		<text>�������� (Russian)</text>
+	</string>
+	<string id="spa">
+		<text>��������� (Spanish)</text>
+	</string>
+	<string id="ukr">
+		<text>��������� (Ukrainian)</text>
+	</string>
+	<string id="jpn">
+		<text>�������� (Japanese)</text>
```

Comment:
Do we have this in other locales? Where did it come from?

Reply:
There is a Japan localization, along with Korean and Chinise... They aren't redistributed officially in Steam, GOG, though.
In fact, I'm not sure if they are official, but it's still nice to left this small addition :)